ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/activity/PaywallActivity.kt

```diff
@@ -43,10 +47,25 @@ internal class PaywallActivity : ComponentActivity(), PaywallViewListener {
     }
 
     private fun getFontProvider(): FontProvider? {
-        return getArgs()?.fonts?.let { fonts ->
-            val fontsMap = fonts.mapValues { entry ->
-                entry.value?.let { fontRes ->
-                    ResourcesCompat.getFont(this, fontRes)?.let { FontFamily(it) }
+        return getArgs()?.fonts?.let { fontFamilyMap ->
+            val googleFontProviders = mutableMapOf<GoogleFontProvider, GoogleFont.Provider>()
```

Comment:
This is a small optimization... Honestly, it's probably not needed, but didn't want to create the provider multiple times.